S2VX.Deploy/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      // Use IntelliSense to find out which attributes exist for C# debugging
+      // Use hover for the description of the existing attributes
+      // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
```

Comment:
JSON doesn't have comments

Reply:
This was auto-generated by the CDK initialization. Gonna leave it since its not causing issues